### DISCUSSION TITLE
How can I close the Popover when mouse moves away?

### DISCUSSION THREAD

**Discussion by darrylmorley** [radix-ui/primitives#2301] [Help] [answered]
Hi,

Can anyone tell me how I might close the Popover element when the mouse is moved away? Like the NavigationMenu component.

**Comment by joaom00** [comment]
You can control the popover with `open` and `onOpenChange` props  and use events to determine when it should close

**Reply by darrylmorley** [reply]
The Navigation component uses onContentLeave, can I use this with Popover somehow?

**Reply by joaom00** [reply, marked as answer]
I don't see how it would be possible
